types/algoliasearch/algoliasearch-tests.ts

```diff
@@ -150,6 +150,7 @@ let _algoliaQueryParameters: QueryParameters = {
   synonyms: true,
   replaceSynonymsInHighlight: false,
   minProximity: 0,
+  sortFacetValuesBy: 'alpha'
```

Comment:
Nit: Can you add a trailing comma after the value so that further diffs will be smaller and don't need to touch this very line?

Reply:
done!